docs/BUILD-PROCESS.md

```diff
@@ -111,4 +111,4 @@ etc. - all `runs` commands inside pipelines run natively on the processor.
 On the other hand, when building for alternative architecture, e.g. for arm64 while on amd64, the commands are run
 using [binfmt_misc](https://en.wikipedia.org/wiki/Binfmt_misc) user-mode emulation.
 
-melange does not need to do anything to make this work, provided `binfmt_misc` is installed on the host system.
+`melange` does not need to do anything to make this work, provided `binfmt_misc` is installed on the host system.
```

Comment:
Why?

Reply:
To me, using backticks for command/package names is good practice and I've seen you do the same in your documents as well. But, if you don't like the backticks or feel that in this case it should be the project name, rather than the executable, then perhaps "Melange" is more correct here?